Core/src/org/sleuthkit/autopsy/coreutils/ImageUtils.java

```diff
@@ -726,8 +753,20 @@ private void saveThumbnail(BufferedImage thumbnail) {
                 if (image.isError() == false) {
                     return image;
                 }
-                //fall through to default image reading code if there was an error
+            } else if (file.getNameExtension().toLowerCase().compareTo("tec") == 0) {
```

Comment:
why not just :
  } else if (file.getNameExtension().equalsIgnoreCase("tec") ) {